src/frontend/apps/desk/src/features/auth/api/getMe.tsx

```diff
@@ -10,6 +10,8 @@ import { fetchAPI } from '@/api';
  */
 export interface UserData {
   email: string;
+  id: string;
+  name: string;
 }
```

Comment:
I am sorry, I don't get why we needed to update this interface?

Reply:
I needed an updated interface because of here:
https://github.com/numerique-gouv/people/pull/73/commits/8267fe3683a10b980d7df4aaf5b02b25f701054d#diff-fec849a8b431e2d0c6feb14982e9e67eb824c62c881dbe902307a2b774d736bcR55

Reply:
Sorry, I totally get it !